substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/pecoff/cv/CVSymbolSubrecord.java

```diff
@@ -246,24 +246,118 @@ public String toString() {
         }
     }
 
+    public static class CVSymbolGData32Record extends CVSymbolSubrecord {
+
+        protected final int typeIndex;
+        protected final int offset;
+        protected final short segment;
+        protected final String name;
+        private final String relativeTo;
+
+        CVSymbolGData32Record(CVDebugInfo cvDebugInfo, short cmd, String name, String relativeTo, int typeIndex, int offset, short segment) {
+            super(cvDebugInfo, cmd);
+            this.name = name;
+            this.relativeTo = relativeTo;
+            this.typeIndex = typeIndex;
+            this.offset = offset;
+            this.segment = segment;
+        }
+
+        @SuppressWarnings("unused")
+        CVSymbolGData32Record(CVDebugInfo cvDebugInfo, String name, int typeIndex, int offset, short segment) {
```

Comment:
Why keep it if it's unused?

Reply:
It was there for future use by references that are not relative.  Removed.